src/ui/camera.js

```diff
@@ -42,15 +42,17 @@ type Required<T> = $ObjMap<T, <V>(v: V) => $NonMaybeType<V>>;
  * camera value for that property will remain unchanged.
  *
  * @typedef {Object} CameraOptions
- * @property {LngLatLike} center The desired center.
+ * @property {LngLatLike} center The location to place at the screen center.
  * @property {number} zoom The desired zoom level.
  * @property {number} bearing The desired bearing in degrees. The bearing is the compass direction that
  * is "up". For example, `bearing: 90` orients the map so that east is up.
  * @property {number} pitch The desired pitch in degrees. The pitch is the angle towards the horizon
- * measured in degrees with a range between 0 and 60 degrees. For example, pitch: 0 provides the appearance
+ * measured in degrees with a range between 0 and 85 degrees. For example, pitch: 0 provides the appearance
  * of looking straight down at the map, while pitch: 60 tilts the user's perspective towards the horizon.
  * Increasing the pitch value is often used to display 3D objects.
- * @property {LngLatLike} around If `zoom` is specified, `around` determines the point around which the zoom is centered.
+ * @property {LngLatLike} around The location serving as the origin for a change in `zoom`, `pitch` and/or `bearing`.
+ * If this location is onscreen, it will remain at the same screen point following the transform.
```

Comment:
Good call!

If the location is offscreen the behavior is the same. So a transform around the location at [-screen_height, -screen_width] will cause that location to remain at the same (offscreen) screen point.

Do you think it would be more clear to change that sentence as follows?

```suggestion
 * This location will remain at the same screen position following the transform.
```

Reply:
Yes I think it's ok! Another useful thing could be trying to answer in which circumstances this argument could be useful.

Reply:
Added a sentence trying to answer that